src/lib.rs

```diff
@@ -101,6 +102,7 @@ pub use bip39;
 pub use bitcoin;
 pub use lightning;
 pub use lightning_invoice;
+pub use payjoin::Uri;
```

Comment:
I added this because our `send` transaction expect  a valid payjoin URI, we can change it to accept a string and run the conversion on our side if we dont want to reexport this